### PR TITLE
fixed minor typo

### PR DESCRIPTION
**Pull request by jmyung**
from #18237

### CODE REVIEW

File: content/en/docs/concepts/services-networking/service.md [hunk 1]
```diff
@@ -887,30 +887,30 @@ There are other annotations for managing Cloud Load Balancers on TKE as shown be
     metadata:
       name: my-service
       annotations:
-        # Bind Loadbalancers with speicfied nodes
+        # Bind Loadbalancers with specified nodes
```
Comment (sftim) [inline content/en/docs/concepts/services-networking/service.md]: Using `git add -p` or equivalent would mean you could propose just the relevant change. If you can do that @jmyung, please do.

Some IDEs that talk Git can also let you selectively commit changes.

Comment (jmyung) [reply content/en/docs/concepts/services-networking/service.md]: Thank you for the tip. @sftim 🎅 
I applied with `git app -p`

File: content/en/docs/concepts/services-networking/service.md [hunk 2]
```diff
@@ -887,7 +887,7 @@ There are other annotations for managing Cloud Load Balancers on TKE as shown be
     metadata:
       name: my-service
       annotations:
-        # Bind Loadbalancers with speicfied nodes
```
Comment (bradtopol) [inline content/en/docs/concepts/services-networking/service.md]: Great catch

**Pull request merged**